lnwallet/channel.go

```diff
@@ -4573,6 +4720,60 @@ func (lc *LightningChannel) ReceiveRevocation(revMsg *lnwire.RevokeAndAck) (
 		}
 	}
 
+	var localPeerUpdates []channeldb.LogUpdate
+
+	// We use the remote commitment chain's tip as it will soon become the tail
+	// once advanceTail is called.
+	remoteMessageIndex := lc.remoteCommitChain.tip().ourMessageIndex
+	localMessageIndex := lc.localCommitChain.tail().ourMessageIndex
+
+	for e := lc.localUpdateLog.Front(); e != nil; e = e.Next() {
+		pd := e.Value.(*PaymentDescriptor)
+
+		if pd.LogIndex < remoteMessageIndex && pd.LogIndex >= localMessageIndex {
+			// This is a settle/fail that is on the remote commitment, but
```

Comment:
Style nit: move the comment to reside _above_ the if statement?

Reply:
Should this always be the case? What if there's an `else`?